doc/decisions/0001-context-in-translations.md

```diff
@@ -0,0 +1,84 @@
+# 1. Context in Translations
+
+## Context
+
+For some translatable strings we create a message context (`msgctxt`) that contains the question text. This enables strings to be translated in the context of the question being asked. There are two current issues with this approach:
+
+- For some elements the context is incorrect. For example, the context for an answer guidance title is "Answer for: {question_text}".
+- Most translatable strings at the question level and below don't have context. This causes problems when the English text doesn't vary between question variants but the translated text needs to - https://trello.com/c/JEdbgtlT/3538-e-translations-add-context-to-all-child-elements-of-a-question-m
+
+
+## Decision
+
+- Remove the current message context `Answer for: {question_text}`.
+- Use a new `Question: {question_text}` message context for all translatable properties within a question other than title. This allows any text within a question to vary based on the question. 
+- Add a new `Type: {property_type}` comment for every string extracted based on the values in the table below e.g. `Type: Question definition list item`. This helps a translator understand what they are being asked to translate. 
+
+  | JSON Path | Type | Notes |
+  |-----------|------|-------|
+  | `$.title` | Questionnaire title | |
+  | `$.legal_basis` | Questionnaire legal basis | |
+  | `$.messages` | Global answer error message | |
+  | `$.sections.[*].title` | Section title | Used for section summaries, final summary and the hub |
+  | `$.sections.[*].repeat.title` | Section title (repeating section) | |
+  | `$.sections.[*].summary.items.[*].title` | Custom section summary item title | |
+  | `$.sections.[*].summary.items.[*].add_link_text` | Custom section summary list add link | |
+  | `$.sections.[*].summary.items.[*].empty_list_text` | Custom section summary empty list text | |
+  | `$..groups.[*].title` | Group title | Used as heading in summaries |
+  | `$..blocks.[*].title` | Block title | Deprecated? |
+  | `$..blocks.[*].summary.title` | List collector summary heading | Deprecated when section summary blocks are removed. |
+  | `$..blocks.[*].summary.item_title` | List collector summary item | Defines how to form the string to summarise each item in the list. Strictly speaking this doesn't need to be translated for all current usages |
```

Comment:
```suggestion
  | `$..blocks.[*].summary.item_title` | List collector summary item | Defines how to form the string to summarise each item in a list. Strictly speaking this doesn't need to be translated for all current usages |
```

Reply:
It's "the" list rather than "a" list as its specifically the list associated with the block this property is attached to.